packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -53,38 +53,46 @@ function PostLink( {
 			onToggle={ onTogglePanel }
 		>
 			{ isEditable && (
-				<TextControl
-					label={ __( 'URL' ) }
-					value={ forceEmptyField ? '' : currentSlug }
-					onChange={ ( newValue ) => {
-						editPermalink( newValue );
-						// When we delete the field the permalink gets
-						// reverted to the original value.
-						// The forceEmptyField logic allows the user to have
-						// the field temporarily empty while typing.
-						if ( ! newValue ) {
-							if ( ! forceEmptyField ) {
+				<div className="editor-post-link">
+					<TextControl
+						label={ __( 'URL Slug:' ) }
```

Comment:
Though my search wasn't exhaustive, it seems as though most input field labels in the sidebar don't end with a colon. The only other one I could see is 'Parent Page:'.

Might be good to drop the colon for sake of consistency. 

@jasmussen what do you think?

Reply:
That's an EXCELLENT catch & eye for detail. Yeah, if we can get that one final detail smoothed out let's ship it. Otherwise, this is also very lovely as is.

Reply:
@talldan @jasmussen removed the colon, nice catch!